.github/workflows/static_code_checks.yml

```diff
@@ -82,6 +82,25 @@ jobs:
 
       - name: Run typechecker
         run: nox -s mypy
+        
+  Fides-Installation:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON_VERSION }}
+
+      - name: Install Nox
+        run: pip install nox>=2022
```

Comment:
It's definitely inefficient to keep doing this in multiple jobs, and I feel like there has to be a more DRY workaround. Do you think this repo is in a place where we could realistically address a follow-up issue to optimize our workflows? We may need to at least block an issue like that on consistently passing tests, but it's the type of thing I'd personally _love_ to address.

Reply:
I don't know a way right off for a DRY workaround, but you can cache the dependencies so they don't have to reinstall on every run. Only one dependency in this case so probably not a big difference, but still some improvement.

Reply:
I have a few ideas, for this and our other workflows. I just don't want to start work on an overhaul until things are stable.

Reply:
@PSalant726 design doc or issue? I'd be really curious to see as well, I agree there could be improvements. Actually in this case, it would easily be a matrix